src/components/Input.scss

```diff
@@ -158,5 +159,39 @@ $form-input-bg-color: $cu-foreground--dark !default;
       @include ss-pika;
     }
   }
+}
 
+// * 5. Errors and feedback
+.Input-feedback {
+  display: none;
+}
+.is-invalid {
+  &.is-required {
+    .Input-label:after {
+      color: $cu-negative;
+    }
+  }
+  .Input-element {
+    border-color: $cu-negative;
+  }
+  .Input-feedback {
+    display: block;
+    color: $cu-negative;
+  }
 }
+// should this be moved to its own scss file since it will also apply to checkboxes/radios/selects?
+// should there be a "forms" component with examples like this and its own scss?
+.is-valid {
+  &.is-required {
+    .Input-label:after {
+      color: $cu-positive;
+      //content: url(assets/icons/check.svg); <- is this folder included in any build path?
+    }
+  }
+  .Input-label:after {
+    color: $cu-positive;
+    display: inline-block;
+    padding-left: $su-xsmall;
+    content: '#'; // just a hash until I get clarity on the svg paths
```

Comment:
I'd use SVG if you can. Not sure how the path generation will work though. We don't have anything built for that and it may get complicated across different app implementations.

Reply:
Used svg in the component rather than using it here in the css. Looking around it seems like that is the better way as SVG referenced in a uri in css can really slow loading. Let me know in the patterns library if you think that is a good way to go for that.